test/verify/check-storage-nfs

```diff
@@ -200,5 +201,62 @@ class TestStorage(StorageCase):
 
         b.wait_present("#detail-header button:contains(Mount)")
 
+@skipImage("NetworkManager and NFS don't work together", "ubuntu-1604")
+class TestStoragePackages(StorageCase, PackageCase):
+
+    def testNfsMissingPackages(self):
+        m = self.machine
+        b = self.browser
+
+        # Override configuration so that we don't have to remove the
+        # real package.
+
+        self.machine.write("/usr/share/cockpit/storaged/override.json",
+                           """{ "config": { "nfs_client_packages": [ "fake-nfs-utils" ] } }""")
+
+        m.execute("mv /sbin/mount.nfs /sbin/mount.nfs.off")
+
+        self.login_and_go("/storage")
+
+        # The fake-nfs-utils package is not available yet
+
+        b.wait_present("button:contains('Install NFS Support')")
+        b.click("button:contains('Install NFS Support')")
+
+        b.wait_in_text("#nfs-mounts", "Support is not available.")
+        b.click("a:contains(View details)")
+        self.dialog_wait_open()
+        b.wait_in_text("#dialog", "fake-nfs-utils is not available from any repository.")
+        self.dialog_cancel()
+        self.dialog_wait_close()
+
+        # Now make the package available
+
+        self.createPackage("dummy", "1.0", "1")
+        self.createPackage("fake-nfs-utils", "1.0", "1", depends="fake-libnfs")
+        self.createPackage("fake-libnfs", "1.0", "1")
+        self.enableRepo()
+
+        # HACK
+        #
+        # The first simulated install seems to silently not report
+        # anything on the Debian test images, for unknown reasons.  So
+        # we install a dummy package to warm up all parts of the
+        # machinery and distribute the fluids evenly.
```

Comment:
This is a stupendously exact technical explanation what's going on  :rofl: 

Could it perhaps be limited to `if "debian" in m.image or "ubuntu" in m.image`, to ensure we know when it happens elsewhere?

Reply:
Done.